src/zenml/integrations/aws/flavors/sagemaker_orchestrator_flavor.py

```diff
@@ -184,6 +184,7 @@ class SagemakerOrchestratorConfig(
 
     Attributes:
         execution_role: The IAM role ARN to use for the pipeline.
+        scheduler_role: The IAM role ARN to use for the scheduler.
```

Comment:
We had some issues ourselves trying to figure out what this role is used for, and what permissions it needs. Maybe it makes sense to have a little more explanation here?

Reply:
You are absolutely right. A full explanation with the roles might be a bit convoluted here though. I am thinking of adding a link to the correct docs section. WDYT?

Reply:
Yep sounds good, short explanation and link to docs?